projects/frontend/data-pipelines/gui/rebuild-rerun.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2021 VMware, Inc.
```

Comment:
I'm not sure about the copyright statements... 

Do we need them?

Reply:
Turns out we don't need them in every file if there is a LICENSE file included. But we can still have them if we really want to.

https://softwareengineering.stackexchange.com/questions/317041/should-i-add-the-license-in-every-header-and-source-file

https://www.apache.org/licenses/LICENSE-2.0#apply

Reply:
We need those, this is the current style we already have consistently with the rest of the repo,
in compliance with OSS requirements 😊

Reply:
We have a plan for the Copyright statements. 

Resolving this discussion.